README.md

```diff
@@ -37,3 +37,14 @@ cd ui
 npm install
 npm start
 ```
+
+# Java Codegen
```

Comment:
I'm not sure we should advertise this in this repo as we don't need nor use `Java` codegen as part of this application.